packages/excalidraw/element/types.ts

```diff
@@ -176,7 +176,6 @@ export type ExcalidrawTextElement = _ExcalidrawElementBase &
     fontSize: number;
     fontFamily: FontFamilyValues;
     text: string;
-    baseline: number;
```

Comment:
@dwelle any special way of handling these deprecations - other than mentioning it in the changelog?